helix-view/src/editor.rs

```diff
@@ -248,6 +251,15 @@ impl Editor {
             return;
         }
 
+        let true_color = self.config.true_color_override
+            || std::env::var("COLORTERM")
+                .map(|v| v == "truecolor" || v == "24bit")
+                .unwrap_or(false);
```

Comment:
This logic here is duplicated, we should share it in a single utility function

Reply:
Where should the utility function go?